securedrop-workstation-config/debian/securedrop-workstation-config.install

```diff
@@ -1,5 +1,6 @@
 mimeapps.list.sd-viewer opt/sdw/
 mimeapps.list.sd-app opt/sdw/
+mimeapps.list.sd-app opt/sdw/mimeapps.list.default
```

Comment:
Based on local testing, this will create a folder `/opt/sdw/mimeapps.list.default/` and copy mimeapps.list.sd-app in that folder.

Reply:
Thanks, good catch. Pulled in your commit, added another, and built the package: https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/70